homeassistant/components/nest/legacy/sensor.py

```diff
@@ -49,6 +53,8 @@
 
 SENSOR_DEVICE_CLASSES = {"humidity": SensorDeviceClass.HUMIDITY}
 
+SENSOR_STATE_CLASSES = {"humidity": SensorStateClass.MEASUREMENT}
+
```

Comment:
It may make sense to use a class like

```
@dataclasses.dataclass
class NestSensorDescription:
    unit: str
    device_class: SensorDeviceClass
    state_class: SensorStateClass

SENSORS = {
    "humidity": NestSensorDescription(unit=PERCENTAGE,
         device_class=SensorDeviceClass.HUMIDITY,
         state_class=SensorStateClass.MEASUREMENT),
}
```

However, the change is correct.